src/main/target/NAZE/target.h

```diff
@@ -193,5 +194,5 @@
 #define USED_TIMERS         (TIM_N(1) | TIM_N(2) | TIM_N(3) | TIM_N(4))
 
 #define TIMER_APB1_PERIPHERALS (RCC_APB1Periph_TIM2 | RCC_APB1Periph_TIM3 | RCC_APB1Periph_TIM4)
-#define TIMER_APB2_PERIPHERALS (RCC_APB2Periph_TIM1 | RCC_APB2Periph_GPIOA | RCC_APB2Periph_GPIOB)
+#define TIMER_APB2_PERIPHERALS (RCC_APB2Periph_TIM1 | BEEPER_GPIO_CLK_PERIPHERAL | RCC_APB2Periph_GPIOB)
```

Comment:
Not sure if the timers for the beeper and other peripherals are in timerHardware yet for the NAZE target, and I wasn't sure how to add them.